src/dotnet-svcutil/lib/tests/Baselines/TypeReuse/TypeReuseDefault/ServiceReference/Reference.cs

```diff
@@ -7,12 +7,12 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
-namespace TypeReuse60_NS
```

Comment:
I see that you are making this generic.  How does this impact combability?  

In addition, perhaps we should throw an erro message when working with version less than 5.0, what do you think?

Reply:
Does generic will make it work with all versions of target Framework?  We do need to think about this.